pkg/tsdb/influxdb/flux/executor.go

```diff
@@ -32,6 +33,7 @@ func ExecuteQuery(ctx context.Context, query QueryModel, runner queryRunner, max
 		dr.Frames = append(dr.Frames, metaFrame)
 		return
 	}
+	defer tables.Close()
```

Comment:
Shouldn't this be before the error check? In case of an error the tables won't be closed.

Reply:
@coussej No, I don't think that would be a good idea, because if an error happened, you can expect the non-error return value to be invalid (a nil pointer in this case).

Reply:
the defer statement will run immediately before the `executeQuery` returns

Reply:
Yes my bad! Ignore my comment.